### PR TITLE
Add relevant massive actions on main form

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: inc/commondbtm.class.php [hunk 1]
```diff
@@ -3792,6 +3792,44 @@ function getForbiddenStandardMassiveAction() {
...
+         if ($ic->getFromDBforDevice($this->getType(), $this->fields['id'])) {
+            $excluded[] = 'Infocom:activate';
+         }
+      }
+
+      return $excluded;
+   }
+
+   /**
+    * Get forbidden single action
```
Comment (cedric-anne) [inline inc/commondbtm.class.php]:
```suggestion
    * Get whitelisted single action
```

File: inc/commondbtm.class.php [hunk 2]
```diff
@@ -3792,6 +3792,44 @@ function getForbiddenStandardMassiveAction() {
...
+    *
+    * @since 9.5.0
+    *
+    * @return array
+   **/
+   public function getWhitelistedActions() {
```
Comment (cedric-anne) [inline inc/commondbtm.class.php]:
```suggestion
   public function getWhitelistedSingleMassiveActions() {
```

File: inc/commondbtm.class.php [hunk 3]
```diff
@@ -3792,6 +3792,44 @@ function getForbiddenStandardMassiveAction() {
...
 
 
+   /**
+    * Get forbidden single action
+    *
+    * @since 9.5.0
+    *
+    * @return array
+   **/
+   public function getForbiddenSingleMassiveAction() {
```
Comment (cedric-anne) [inline inc/commondbtm.class.php]:
```suggestion
   public function getForbiddenSingleMassiveActions() {
```

**Comment by orthagh** [comment]
PR up to date, ready for a final review and merge if ok

**Comment by trasher** [comment]
Just tested, ok for me :)

**Pull request merged**